package.json

```diff
@@ -54,8 +53,8 @@
     "@angular-devkit/build-angular": "~0.13.0",
     "@angular/language-service": "^7.2.3",
     "@types/file-saver": "^1.3.0",
-    "@types/node": "^10.5.2",
     "@types/jest": "^23.1.1",
+    "@types/node": "^10.5.2",
```

Comment:
I see that this was here before, but I'm not sure we need this. The app isn't running inside node.